experimental/PropertyDDS/packages/property-dds/src/propertyTree.ts

```diff
@@ -437,7 +439,7 @@ export class SharedPropertyTree extends SharedObject {
 			snapshot.numChunks = chunks.length;
 		}
 
-		return {
+		return convertToSummaryTreeWithStats({
```

Comment:
Not sure if you want to add it to this PR, but glancing through the code, we can remove fullTree argument in convertToSummaryTreeWithStats() / convertToSummaryTree() - ITree here will never have id. Even better would be to probably follow up (with a set of separate issues) to remove all ITree-related code from DDSs and use directly summary interfaces (and kill ITree -> summary conversion functions)

Reply:
I opened issue #8652